packages/shades/shades.1.0/opam

```diff
@@ -0,0 +1,37 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml color palettes"
+description: "A collection of R color palettes reformatted for OCaml"
+maintainer: ["faycarsons23@gmail.com"]
+authors: ["Fay Carsons"]
+license: "MIT"
+homepage: "https://github.com/FayCarsons/ocaml-shades"
+bug-reports: "https://github.com/FayCarsons/ocaml-shades/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.12"}
+  "odoc"
```

Comment:
```suggestion
  "odoc" {with-doc}
```
Odoc is not directly used in the project, just for the documentation.

If you remove odoc from the list of dependencies, it will be fixed for the future: https://github.com/FayCarsons/ocaml-shades/commit/12876b237ab2032b3158737ca856730342483051#diff-1fb9d92d4f0b89e8c93cdefc87d39436b213422961c40453cf2fef5f4a195c57R19